setup/src/setup.cpp

```diff
@@ -18,7 +19,7 @@ namespace fs = std::filesystem;
 void copyVideos(std::wstring videoDir, std::string outputDir) {
     for (const fs::directory_entry& entry : fs::directory_iterator(videoDir)) {
         const fs::path path = entry.path();
-        std::filesystem::copy(path, outputDir + "\\" + path.filename().string());
+        std::filesystem::copy(path, outputDir + "\\" + path.filename().string(), std::filesystem::copy_options::update_existing );
```

Comment:
Can we split this over multiple lines?

I try to keep to an 80-character limit where possible.

    std::filesystem::copy(path, outputDir + "\\" + path.filename().string(),
        std::filesystem::copy_options::update_existing );